### DISCUSSION TITLE
Is there a way to not allow editors to add new documents to a collection?

### DISCUSSION THREAD

**Discussion by raguila8** [tinacms/tinacms#2297] [Help] [answered]
Specifically, the "Add document" modal has a "Choose collection" select input menu with every defined collection as an option. Is there a way to remove a collection from this list? The tina-cloud-starter template does this and doesn't show the "Pages" collection. I can't figure out how it's doing this. There doesn't seem to be a flag for this. Here is a snippet from the tina-cloud-starter that only shows the Blog Posts collection and excludes the Pages collection:

![theone](https://user-images.githubusercontent.com/29444224/143973679-b69c576a-390c-436c-97a0-f2573c62d750.gif)

How was this done? I can't replicate this behavior on my project.

**Comment by perkinsjr** [comment, marked as answer]
Hey there! 

Thanks for reaching out, you can see an example in the code here https://github.com/tinacms/tina-cloud-starter/blob/main/pages/_app.js 

The basic premise is to set the filterCollections property with the allowed creation places

```
documentCreatorCallback={{
              /**
               * After a new document is created, redirect to its location
               */
              onNewDocument: ({ collection: { slug }, breadcrumbs }) => {
                const relativeUrl = `/${slug}/${breadcrumbs.join("/")}`;
                return (window.location.href = relativeUrl);
              },
              /**
               * Only allows documents to be created to the `Blog Posts` Collection
               */
              filterCollections: (options) => {
                return options.filter(
                  (option) => option.label === "Blog Posts"
                );
              },
            }} ```

Hope this helps!

**Reply by raguila8** [reply]
Gotcha! Thanks, I was looking in the schema file.
